Description: Remove debian dependency files from built python source package
When we've built the original source payload requirements.txt should be the
same as the intermediary file debian-requirements.txt, and deb-dependencies.txt
is just used for installing build packages in a dev enviroment.

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- conn-check-1.0.13.orig/MANIFEST.in
+++ conn-check-1.0.13/MANIFEST.in
@@ -1,5 +1,7 @@
 include LICENSE
 include CHANGELOG
 include *.txt
+exclude debian-requirements.txt
+exclude deb-dependencies.txt
 recursive-include conn_check *.txt
 recursive-include conn_check *.xml
